blackbox/docs/src/doc_tests/process_test.py

```diff
@@ -337,7 +347,35 @@ def tearDown(self):
         self.clients[0].sql('DROP TABLE t1')
 
     @staticmethod
-    def exec_queries(client, is_active, errors, finished):
+    def exec_insert_queries(client, is_active, errors, finished):
+        while is_active[0]:
+            try:
+                chars = list(string.ascii_lowercase[:14])
+                random.shuffle(chars)
+                client.sql(
+                    'insert into t1 (id, name) values ($1, $2) on duplicate key update name = $2',
+                    (random.randint(0, 2147483647), ''.join(chars))
+                )
+            except Exception as e:
+                errors.append(e)
```

Comment:
We don't need to catch the `TableUnknownException` here?

Reply:
I ran it for quite a while and it didn't fail. I think the internal retry logic in the transport base classes take care of that.